proto/crosschain/query.proto

```diff
@@ -253,6 +257,14 @@ message QueryAllPendingNoncesResponse {
   repeated PendingNonces pending_nonces = 1;
 }
 
+message QueryPendingNoncesByChainRequest {
+  uint64 chain_id = 1;
+}
+
+message QueryPendingNoncesByChainResponse {
+  PendingNonces pending_nonces = 1 [(gogoproto.nullable) = false];
+}
+
```

Comment:
Not a very big deal but maybe we might want to keep the same order for message definitions as rpc endpoint definitions above